torch/csrc/jit/passes/quantization.cpp

```diff
@@ -373,7 +336,7 @@ void InsertQuantDequantNodes(
   blocks_to_visit.push(graph->block());
   // For storing quantizable values - node pairs that are external
   // or intermediate inputs to quantizable nodes
-  std::vector<std::pair<Value*, Node*>> quantInputs;
+  std::vector<param_info_t> quantInputs;
```

Comment:
This change seems unrelated.

Reply:
This is one of ur older comment to use struct instead of pair . Its used below since we have struct defined already

Reply:
I'd rather prefer it to be addressed in the PR where I commented and not here, but that's also fine :)

Reply:
i'm also fine with this change in this PR.